tests/common.rs

```diff
@@ -80,6 +80,43 @@ macro_rules! expect_channel_ready_event {
 
 pub(crate) use expect_channel_ready_event;
 
+macro_rules! expect_payment_received_event {
```

Comment:
@tnull I added the following macros following the pattern in the file. for some reason, the macros I added are marked as "unused" although they are used in the multihop test. what am i missing?
Also, should I split the commit for adding those two, or its ok to be in the same commit?

Reply:
"unused_macros: unused macro definition: `expect_payment_successful_event`"